.eslintignore

```diff
@@ -1 +1,3 @@
 app/dist
+/migrations/migrations/20190626181121-users.js
+app/models/user.js
```

Comment:
Remove this from ignore, this files should follow best practices too because are part of code